src/Microsoft.Azure.WebJobs.Logging.ApplicationInsights/Initializers/SdkVersionTelemetryInitializer.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.ApplicationInsights.Channel;
+using Microsoft.ApplicationInsights.DataContracts;
+using Microsoft.ApplicationInsights.Extensibility;
+using Microsoft.ApplicationInsights.Extensibility.Implementation;
+
+namespace Microsoft.Azure.WebJobs.Logging.ApplicationInsights
+{
+    internal class SdkVersionRequestTelemetryInitializer : ITelemetryInitializer
```

Comment:
I'm having second thoughts on this now (even though I suggested it). This means that Functions/CLI will need to register an additional initializer to overwrite this one, which feels clunky.

However, I think it's fine for this initial commit, and when I pull it through to Functions, I may come back and revisit. Even having an extra class like an ISdkVersionProvider that we could register once and use in different places may be a better fit. Then each place we'd register our own and trust that it'd be set up correctly everywhere.